### PR TITLE
add: Added 2 pages.

### PR DESCRIPTION
**Pull request by jainvedant392**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,22 +3,22 @@
...
+* Add different clothing items.
+* Select pickup date and location at convenience.
+* Make secure online payments.
+* View and review your past orders.
 <Br><Br>
 
 # <img src="https://github.com/NightFury742/LaundriX/assets/119070798/28a131bf-d93f-44a2-b368-7eaa1476685b" height="30px"> Design
 
-[Checkout our website design](https://www.figma.com/file/Yq77JsE5rNfOIuwUEYgqtr/Laundrix?type=design&t=q4XGVlCGgkAkNEar-6)
+[Checkout our Figma project design](https://www.figma.com/file/Yq77JsE5rNfOIuwUEYgqtr/Laundrix?type=design&t=q4XGVlCGgkAkNEar-6)
```
Comment (NightFury742) [inline README.md]: Instead of Firgma project just write Figma design

File: README.md [hunk 2]
```diff
@@ -28,31 +28,31 @@ Users will be able to book their preferred service, select convenient time slots
 * Chakra UI
 
 ## Backend
-* FireStone
+* Google Firebase
```
Comment (NightFury742) [inline README.md:31]: Let it be FireStone

File: README.md [hunk 3]
```diff
@@ -28,31 +28,31 @@ Users will be able to book their preferred service, select convenient time slots
...
 
 ## Others
-* Firebase
 * Framer Motion
-* Lottie
+* Lottie-react
```
Comment (NightFury742) [inline README.md:35]: Please make it Lottie

File: README.md (hunk 1 above)
Comment (jainvedant392) [reply README.md]: Sure @NightFury742, will do the same.

**Comment by jainvedant392** [comment]
Sure @NightFury742, will look into all the changes requested.

**Comment by jainvedant392** [comment]
I have done the changes. PTAL @NightFury742 .

**Pull request merged**